src/sdp/parser.rs

```diff
@@ -1210,7 +1219,7 @@ mod test {
     }
 
     #[test]
-    fn parse_candidate_ufrag() {
+    fn parse_candidates() {
```

Comment:
renaming the test as I see the existing tests also assert more than just ufrag, so I'm adding tcp assertions here as well.